src/components/NftPreview/GalleryNftPreviewWrapper.tsx

```diff
@@ -55,34 +54,7 @@ function GalleryNftPreviewWrapper({ tokenRef, columns }: Props) {
   const isMobile = useIsMobileWindowWidth();
   const previewSize = isMobile ? MOBILE_NFT_WIDTH : LAYOUT_DIMENSIONS[columns];
 
-  const { aspectRatioType } = useContentState();
-
-  const nftPreviewMaxWidth = useMemo(() => {
-    if (columns > 1) return '100%';
-
-    // this could be a 1-liner but wanted to make it explicit
-    if (columns === 1) {
-      if (isMobile) {
-        return '100%';
-      }
-      if (aspectRatioType === 'wide') {
-        return '100%';
-      }
-      if (aspectRatioType === 'square' || aspectRatioType === 'tall') {
-        return '60%';
```

Comment:
@jakzaizzat getting rid of the logic in this PR

Reply:
Goit it